src/config/config.js

```diff
@@ -0,0 +1,34 @@
+const Sequelize = require('sequelize')
+const Op = Sequelize.Op
+
+require('dotenv').config()
+
+module.exports = {
+  development: {
+    url: process.env.DATABASE_URL,
+    dialect: 'postgres',
+    operatorsAliases: {
+      $and: Op.and,
+      $or: Op.or,
+      $eq: Op.eq,
+      $gt: Op.gt,
+      $lt: Op.lt,
+      $lte: Op.lte,
+      $like: Op.like
+    }
+  },
+  test: {
+    username: 'root',
+    password: null,
+    database: 'database_test',
+    host: '127.0.0.1',
```

Comment:
please put this in the env file. If you can I would love to use sqlite here as the dialect. As testing can be done in memory.

Reply:
@rotimi-babalola  this?